reference/docs-conceptual/getting-started/fundamental/Learning-PowerShell-Names.md

```diff
@@ -0,0 +1,141 @@
+---
+ms.date:  08/24/2018
+keywords:  powershell,cmdlet
+title:  Learning PowerShell names
+ms.assetid:  b4d0fd22-8298-4ee6-82ae-9b6f2907c986
+---
+
+# Learning PowerShell names
+
+Learning names of commands and parameters requires a significant time investment with most
+command-line interfaces. The issue is that there are few patterns. Memorization is only way to
+learn the commands and parameters that you need to use on a regular basis.
+
+When you work with a new command or parameter, you can't always use what you already know. You have
+to find and learn a new name. Traditionally, command-line interfaces start with a small set of
+tools and grow with incremental additions. Tt is easy to see why there's no standard structure.
+This seems logical for command names since each command is a separate tool. PowerShell has a better
+way to handle command names.
+
+## Learning command names in traditional shells
+
+Most commands are built to manage elements of the operating system or applications, such as
+services or processes. The commands have names that may or may not fit into a family. For example,
+on Windows systems, you can use the `net start` and `net stop` commands to start and stop a
+service. **SC.EXE** is another service control tool for Windows. That name does not fit into the
+naming pattern for the `net` service commands. For process management, Windows has the `tasklist`
+command to list processes and the `taskkill` command to kill processes.
+
+Also, these commands have irregular parameter specifications. You can't use the `net start` command
+to start a service on a remote computer. The `sc` command can start a service on a remote computer.
```

Comment:
To the previous point, now sc is in a code-fence and without the .exe and without bold.

Reply:
It is code fenced here because it is the command form of usage.